src/Documents.jl

```diff
@@ -189,6 +189,7 @@ immutable User
     linkcheck_ignore::Vector{Union{String,Regex}}  # ..and then ignore (some of) them.
     checkdocs::Symbol         # Check objects missing from `@docs` blocks. `:none`, `:exports`, or `:all`.
     strict::Bool              # Throw an exception when any warnings are encountered.
+    pattern_ignore::Nullable{Regex}  # Ignore differences that match this pattern in doctests.
```

Comment:
Should we make it an array perhaps, so that one could have multiple patterns? Should we call it `doctest_ignore`, to namespace it more clearly?